.github/ISSUE_TEMPLATE/not-meeting.yml

```
@@ -0,0 +1,18 @@
+name: Group not meeting
+description: Let the W3C staff know your group is not planning to meet.
+labels: ["not-meeting"]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Please use this form to let the W3C staff know your group is not planning to meet during TPAC. The title should be the name of the group. The name must be the official [group name in the W3C system](https://www.w3.org/groups/), with some abbreviations supported (e.g., WG for Working Group). 
+
+- type: textarea
+    id: comments
+    attributes:
+      label: Comments for the W3C staff (Optional)
+      description: |
+        If you wish to share any information with the W3C staff about your group's decision not to meet during TPAC, please use the space below.

```

Description may be misleading proposers into thinking they are only sharing information with the W3C staff, whereas information is going to be public on GitHub.

```suggestion
        If you wish to share any information about your group's decision not to meet during TPAC, please use the space below.
```